.travis.yml

```diff
@@ -20,3 +20,4 @@ node_js:
 
 script:
     - npm test
```

Comment:
lets add back `gulp lint` while we're at it so we know things are being linted accordingly.